variants/BharatPi-Node-Wifi/pins_arduino.h

```diff
@@ -0,0 +1,51 @@
+#ifndef Pins_Arduino_h
+#define Pins_Arduino_h
+
+#include <stdint.h>
+
+static const uint8_t LED_BUILTIN = 2;
+#define BUILTIN_LED LED_BUILTIN  // backward compatibility
+#define LED_BUILTIN LED_BUILTIN  // allow testing #ifdef LED_BUILTIN
+
+static const uint8_t SAFFRON_LED  = 12;
+static const uint8_t WHITE_LED = 2;
+static const uint8_t GREEN_LED = 13;
+
+static const uint8_t A0 = 14;
+static const uint8_t A1 = 13;
+static const uint8_t A2 = 12;
+static const uint8_t A3 = 4;
+static const uint8_t A4 = 2;
+static const uint8_t A5 = 0;
+
+static const uint8_t TX = 1;
+static const uint8_t RX = 3;
+
+static const uint8_t TX2 = 17;
+static const uint8_t RX2 = 16;
+#endif /* Pins_Arduino_h */#ifndef Pins_Arduino_h
+#define Pins_Arduino_h
+
+#include <stdint.h>
+
+static const uint8_t LED_BUILTIN = 2;
+#define BUILTIN_LED LED_BUILTIN  // backward compatibility
+#define LED_BUILTIN LED_BUILTIN  // allow testing #ifdef LED_BUILTIN
+
+static const uint8_t SAFFRON_LED  = 12;
+static const uint8_t WHITE_LED = 2;
+static const uint8_t GREEN_LED = 13;
+
+static const uint8_t A0 = 14;
+static const uint8_t A1 = 13;
+static const uint8_t A2 = 12;
+static const uint8_t A3 = 4;
+static const uint8_t A4 = 2;
+static const uint8_t A5 = 0;
+
+static const uint8_t TX = 1;
+static const uint8_t RX = 3;
+
+static const uint8_t TX2 = 17;
+static const uint8_t RX2 = 16;
+#endif /* Pins_Arduino_h */
```

Comment:
This part is doubled, please remote it.
```suggestion
#endif /* Pins_Arduino_h */
```